README.md

```diff
@@ -285,6 +286,17 @@ existing fidoconfig file if you have one.
    othernets do not (regularly) distribute a nodediff file, but just hatch a
    new nodelist every few weeks.  In this case, you need the FullUpdate
    statement.
+```
+   Dailynodelist
+   ----------
+   Syntax:  Dailynodelist <On/Off>
+   Example: Dailynodelist On
+```
+   This statement tells to nlupdate to process a daily nodelist, you have
+   to set fullupdate and dailynodelist statements and not the diffupdate 
```

Comment:
to set fullupdate and dailynodelist statements and not the diffupdate => to set "FullUpdate" and "Dailynodelist" statements and not the "DiffUpdate"